gtsfm/bundle/bundle_adjustment.py

```diff
@@ -275,14 +353,26 @@ def __cameras_to_model(
         proxy for this function."""
         cameras: Set[int] = set(initial_data.get_valid_camera_indices())
 
+        # Add all relative pose priors have atleast one node in the GTSFM's data valid cameras.
+        for i1, i2 in relative_pose_priors.keys():
+            if i1 in cameras or i2 in cameras:
+                cameras.add(i1)
+                cameras.add(i2)
+
+        # Hilti specific hack to add the backbone of CAM2-CAM2
```

Comment:
What do we do here?

Reply:
This is a hack we did to add all CAM2-CAM2 edges. But I think its not needed anymore because of changes in loader. Will remove this.

Reply:
Removed it.